src/SQLQueryStress/SQLQueryStress.csproj

```diff
@@ -39,7 +39,7 @@
     <ProductName>SQLQueryStress</ProductName>
     <PublisherName>Adam Machanic</PublisherName>
     <ApplicationRevision>0</ApplicationRevision>
-    <ApplicationVersion>0.9.0.0</ApplicationVersion>
+    <ApplicationVersion>0.10.0.0</ApplicationVersion>

```

Comment:
Please revert this, the build server increments the build number

Reply:
done